apps/functions-subscription/src/adapters/azure/functions/__tests__/create-trial.test.ts

```diff
@@ -8,15 +8,15 @@ import { makePostTrialHandler } from '../create-trial';
 import { ItemAlreadyExists } from '../../../../domain/errors';
 
 describe('makePostTrialHandler', () => {
-  it('should return 201 with the created trial', async () => {
+  it('should return 202 with the id and the state of the trial', async () => {
```

Comment:
**nitpick**: The test says `with the id and the state of the trial`, but the return structure is not tested